pkg/types/conversion/installconfig.go

```diff
@@ -26,19 +27,21 @@ func ConvertInstallConfig(config *types.InstallConfig) error {
 	default:
 		return field.Invalid(field.NewPath("apiVersion"), config.APIVersion, fmt.Sprintf("cannot upconvert from version %s", config.APIVersion))
 	}
-	ConvertNetworking(config)
+	convertNetworking(config)
 
 	switch config.Platform.Name() {
 	case baremetal.Name:
-		ConvertBaremetal(config)
+		convertBaremetal(config)
+	case openstack.Name:
+		convertOpenStack(config)
 	}
 
 	config.APIVersion = types.InstallConfigVersion
 	return nil
 }
 
-// ConvertNetworking upconverts deprecated fields in networking
-func ConvertNetworking(config *types.InstallConfig) {
+// convertNetworking upconverts deprecated fields in networking
+func convertNetworking(config *types.InstallConfig) {
```

Comment:
I dunno why this was public before, but making it private seems like an orthogonal change that should go in its own PR.

Reply:
okay, I'll propose another PR to fix this.